tests/foreman/api/test_docker.py

```diff
@@ -108,7 +112,7 @@ def _create_repository(product, name=None, upstream_name=None):
 
     """
     if name is None:
-        name = gen_string(gen_choice(STRING_TYPES), 15)
+        name = generate_strings_list(15, ['numeric', 'html'])
```

Comment:
Oops. It is actually :) but do we have any specific reason to exclude these types?